MewsSystems/mews-flutter
# feat(storybook): Updated storybook version. Added saving theme.

#### Summary

Updated storybook dependency. Added theme saving.

#### Testing steps

1. Open Storybook.
2. Change the theme.
3. Hot-reload - theme should be saved.

You can test it on the firebase [here](https://witwash-storybook.web.app/#/).

#### Follow-up issues

None.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
